src/styles/PopUpModal.css

```diff
@@ -40,14 +40,13 @@
         transition-duration: 0.2s;
         transition-timing-function: cubic-bezier(0.4, 0, 0.2, 1);
 
-        overflow-y: auto;
         border-radius: 16px;
         -ms-scroll-chaining: none;
         overscroll-behavior: contain;
 
         position: absolute;
         top: 10%;
-        width: 450px;
+        max-width: 450px;
```

Comment:
shouldn't there be a width or min-width? The modal is used in a lot of places, it could look inconsistent if the size varies too much

Reply:
You're right! I added a min-width